bodhi/server/scripts/dequeue_stable.py

```diff
@@ -0,0 +1,40 @@
+# This program is free software; you can redistribute it and/or
```

Comment:
According to Tony Sebro, you get to hold the copyright on this code (neat, huh? In most places, your employer would hold the copyright on code that you are paid to write so that's pretty cool I think). So you should do this in this file and the test file you wrote:

```
# -*- coding: utf-8 -*-
# Copyright © 2017 Caleigh Runge-Hottman
#
# This file is part of Bodhi.
#
<the rest of the file as you have it, including the GPL stuff>
```